Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -396,8 +397,21 @@ function Invoke-CodeGen {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         $tool = (Join-Path $toolsPath 'EdFi.Ods.CodeGen')
         $repositoryRoot = (Get-RepositoryRoot $implementationRepo).Replace($implementationRepo, '')
+                
+        if (((-not [string]::IsNullOrWhiteSpace($ExtensionLocationPaths)) -and (-not $IncludePlugins))) {
+            
+            $extensionLocationPlugins = @()
+            Foreach ($extensionLocationPath in $ExtensionLocationPaths) {
+                if (![System.IO.Directory]::Exists($extensionLocationPath)) {
+                    throw "Unable to find extension Location project path at location $extensionLocationPath ."
+                }
+
+                $extensionLocationPlugins += $extensionLocationPath
+            }
 
-        if ($IncludePlugins) {
+            & $tool -r $repositoryRoot -e $Engine -j $extensionLocationPlugins | Write-Host
+        }
+        elseif ($IncludePlugins) {
```

Comment:
again this is an independent statement, the use case is that an end user could accept the plugin for tpdm and add their own extension

Reply:
yes accept both as well